src/core/diags/factory_diags.cpp

```diff
@@ -413,8 +414,8 @@ Error Diags::ProcessRepeat(uint8_t aArgsLength, char *aArgs[])
                      error = kErrorInvalidArgs);
         mTxLen = txLength;
 
-        mRepeatActive = true;
-        uint32_t now  = otPlatAlarmMilliGetNow();
+        mCurTxCmd    = kTxCmdRepeat;
+        uint32_t now = otPlatAlarmMilliGetNow();
```

Comment:
Let's move local variable `now` declaration to the top of scope (to follow style guide)?
(looks like this was existing code, but may be good to fix it in this PR as well?)

Reply:
Or actually , we can remove it fully and change 
```c++
   otPlatAlarmMilliStartAt(&GetInstance(), otPlatAlarmMilliGetNow(), mTxPeriod);
```

 What do you think?